doc/news/changes/incompatibilities/20181008Fehling

```diff
@@ -0,0 +1,6 @@
+Changes: The previously called function
```

Comment:
I thought I had commented on this -- or was that a different PR?

Reply:
Ha! It was the other PR. Let me fix this quickly...